pkg/webhook/playground_webhook.go

```diff
@@ -122,5 +123,21 @@ func (w *PlaygroundWebhook) generateValidate(obj runtime.Object) field.ErrorList
 			allErrs = append(allErrs, field.Forbidden(specPath.Child("modelClaims", "models"), "only one main model is allowed"))
 		}
 	}
+
+	if playground.Spec.BackendRuntimeConfig != nil && playground.Spec.BackendRuntimeConfig.Resources != nil {
+		requirements := playground.Spec.BackendRuntimeConfig.Resources
+		for resourceName, limit := range requirements.Limits {
+			request, ok := requirements.Requests[resourceName]
+			if !ok {
+				continue
+			}
+
+			if limit.Cmp(request) < 0 {
```

Comment:
Let's be more clearly here:
```suggestion
			if limit.Cmp(request) == -1 {
```

Reply:
done